test/scenarios/bookshop/funcs.test.js

```diff
@@ -249,14 +276,16 @@ describe('Bookshop - Functions', () => {
       expect(res.data.value.length).to.be.eq(1)
     })
     test.skip('maxdatetime', async () => {
-      // REVISIT: ERROR: Feature is not supported: Method "maxdatetime" in $filter or $orderby query options
+      // okra error: Feature is not supported: Method "maxdatetime" in $filter or $orderby query options
+      // new adapter error: 500 - sql error
```

Comment:
The `maxdatetime` and `mindatetime` are missing the SQL string quotes.

[maxdatetime](https://github.com/cap-js/cds-dbs/blob/main/db-service/lib/cql-functions.js#L203)